.github/workflows/ci.yml

```diff
@@ -2027,8 +2027,10 @@ jobs:
         id: get-coverage-reports
         uses: actions/download-artifact@v4
         with:
-          name: all-testrun-coverage-artifacts
+          ## name: all-testrun-coverage-artifacts
```

Comment:
Remove commented out name?

Reply:
It was wip since the very final combine testrun failed, wanted to check fix